lib/ansible/modules/system/authorized_key.py

```diff
@@ -13,81 +13,86 @@
                     'supported_by': 'core'}
 
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: authorized_key
 short_description: Adds or removes an SSH authorized key
 description:
-    - "Adds or removes SSH authorized keys for particular user accounts"
+    - Adds or removes SSH authorized keys for particular user accounts.
 version_added: "0.5"
 options:
   user:
     description:
-      - The username on the remote host whose authorized_keys file will be modified
+      - The username on the remote host whose authorized_keys file will be modified.
+    type: str
     required: true
   key:
     description:
-      - The SSH public key(s), as a string or (since 1.9) url (https://github.com/username.keys)
+      - The SSH public key(s), as a string or (since Ansible 1.9) url (https://github.com/username.keys).
+    type: str
     required: true
   path:
     description:
-      - Alternate path to the authorized_keys file
-    default: "(homedir)+/.ssh/authorized_keys"
+      - Alternate path to the authorized_keys file.
+      - When unset, this value defaults to I(~/.ssh/authorized_keys).
```

Comment:
Yes, two reasons. It's not a module default, but a low-level tool default, and it causes a sanity error because the arg_spec doesn't match the documentation.

We have various other cases where defaults cannot be put as a default in the docs and are listed in the description, so it's not that exceptional.

Reply:
That makes sense, thanks for the explanation.